runtime/common/src/lib.rs

```diff
@@ -51,6 +51,23 @@ macro_rules! decl_common_types {
 
         type Address = sp_runtime::MultiAddress<AccountId, ()>;
 
+        #[cfg(feature = "parachain")]
+        pub type Executive = frame_executive::Executive<
+            Runtime,
+            Block,
+            frame_system::ChainContext<Runtime>,
+            Runtime,
+            AllPalletsWithSystem,
+            (
+                pallet_author_mapping::migrations::AddKeysToRegistrationInfo<Runtime>,
```

Comment:
@sea212 could you please provide some context about these new migrations?

Reply:
[`AddKeysToRegistrationInfo` adds a session key to author mapping](https://github.com/PureStake/moonbeam/pull/1407). This session key is used for Verifiable Random Functions (VRF) that can provide an improved source of randomness.
[`AddAccountIdToNimbusLookup` adds a reverse lookup](https://github.com/PureStake/moonbeam/pull/1525) `account_id` -> `nimbus_id` to `pallet_author_mapping` that is required.
[`SplitDelegatorStateIntoDelegationScheduledRequests`](https://github.com/PureStake/moonbeam/pull/1408) does plenty of things, please refer to the link.

Reply:
:+1: